### ISSUE TITLE
wk4 aug_epic: UAT Bug? Coplan did not send (Note: Facil, Admin, Setup Complete are still checked after the nightly run)

### ISSUE THREAD

**Issue by lijenn** [lzim/teampsd#2198]
1. **Who encountered the bug:** Matt Moore

2. **Which instance are you testing in?:** QA

3. **Date/Time of the bug:** Finished the emails on Wed. 8/25 between 1-2pm Pacific. Noticed emails did not send this morning  7am Pacific on Thurs. 8/26.

3. **Role:** Facilitator

4. **Detailed description of the bug:** 

Matt completed steps 6.9 & 6.13.
Checked the next day and noticed the co-plan email did not send.
![image](https://user-images.githubusercontent.com/59668647/130993275-854d702b-2165-4f6b-9efb-311ffac6d82f.png)

@hsnyder2 @apisano0920 Could the nightly routine bug from yesterday for the checked boxes impacted the emails from being sent?

Matt confirmed with me that he did not check the Facilitator, Admin, or Setup complete check boxes. I believe since we met at 9am Pacific yesterday, we had re-checked the Setup complete box to get the co-planning group to show up in the Facilitator's dashboards. 

So Matt would have set up the email to be sent, after we clicked the Setup complete box...would this have prevented his co-planning email from sending?
Note: Even after the nightly routine, I noticed the Facil, Admin, and Setup complete boxes for his group is still checked. Shouldn't they have unchecked itself, since this is a new day?

**Comment by hsnyder2** [comment]
Arlene is adding logic to prevent bad session data links for teams when they're set up.  Will need to retest this when it's deployed to QA.

**Comment by hsnyder2** [comment]
OK, it's possible that this may have also been a result of this observation by Arlene:
I think I figured out why some of the nightly jobs weren't running. There were 3 that were not running. All 3 were owned by Chad. He was set up as an MTL Facilitator. I reassigned all of his ownership settings with the new Application Account User. We'll see if they run tonight.
She's swapped out Chad's account that these run under with the Application User Account we just received today.  She'll demo the Session Validation functionality that she's added in our Sprint Review tomorrow.  We expect that, with these two "fixes," there really isn't a scenario that could cause this to happen again.
